src/SDK/Logs/LogRecordExporterInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Logs;
+
+use OpenTelemetry\SDK\Common\Future\CancellationInterface;
+use OpenTelemetry\SDK\Common\Future\FutureInterface;
+
+interface LogRecordExporterInterface
+{
+    /**
+     * @param iterable<ReadableLogRecord> $batch
+     */
+    public function export(iterable $batch, ?CancellationInterface $cancellation = null): FutureInterface;
+    public function forceFlush(?CancellationInterface $cancellation = null): bool;
```

Comment:
I see that these two functions `forceFlush` and `shutdown` are declared in at least few interfaces. `LogRecordProcessorInterface` contains them as well. They are also in other places inside SDK. Maybe we could extract common interface?

Reply:
That's true, I can find 10 interfaces in the SDK that implement both `forceFlush` and `shutdown`, in two different flavours (some have cancellation and some don't).
I guess we could extract those methods into two more general interfaces and have all those other interfaces extend one, but struggling to think of a good name for them.